gammapy/cube/fit.py

```diff
@@ -683,6 +688,66 @@ def to_dict(self, filename=""):
         data["filename"] = filename
         return data
 
+    def to_spectrum_dataset(self, on_region):
```

Comment:
Maybe also add the `containment_correction` argument here?

Reply:
I have added it here. It requires that the `MapDataset` contains a `PSFMap` as `psf` attribute.